lib/temporal/failure_converter/basic.rb

```diff
@@ -41,7 +41,8 @@ def to_failure(error, payload_converter)
 
         failure.message = error.message
         failure.stack_trace = error.backtrace&.join("\n") || ''
-        failure.cause = to_failure(error.cause, payload_converter) if error.cause
+        # RBS: StandardError fallback is only needed to satisfy steep - https://github.com/soutaro/steep/issues/477
```

Comment:
Unrelated..
I just saw we're calling this class FailureConverter::Basic instead of `Default` like the other SDKs.

Reply:
It's pretty weird to call a class itself a default, since it's how you use it that makes it default, not its implementation (hence the `FailureConverter::DEFAULT` constant that points to the basic implementation.

But again, happy to change that in line with other SDKs